revolv/static/project.scss

```diff
@@ -77,3 +510,207 @@
     height: 300px;
     margin: 0 20px 20px 20px;
 }
+
+
+@media screen and (max-width: 1050px) {
+    .project-updates {
+
+        .small-12 {
```

Comment:
You should definitely target this div in a different way, if we change it to take up a smaller number of columns, css will break